src/App.js

```diff
@@ -1,26 +1,45 @@
 import React, { useState } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { Link } from "react-router-dom";
 
-import { Workout } from "./Components/Workout/Workout";
+import Paper from "@material-ui/core/Paper";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import SearchIcon from "@material-ui/icons/Search";
+import CreateIcon from "@material-ui/icons/Create";
 
-import "./App.css";
+// view components
+import ExerciseFinder from "./Views/ExerciseFinder.jsx";
+import RecordWO from "./Views/RecordWO.jsx";
+
+// custom components
+
+// custom styles
+import { ThemeProvider } from "@material-ui/core";
+import theme from "./theme";
 
 const App = () => {
-  const [workouts, setWorkouts] = useState([
-    { name: "Squats", description: "Squats description example" },
-    { name: "Deadlifts", description: "Deadlifts description example" },
-    { name: "Lunges", description: "Lunges description example" },
-  ]);
+  const [value, setValue] = React.useState(0);
 
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
   return (
-    <div className="App">
-      <h1>React Workout App</h1>
-      <div className="Workouts">
-        {workouts.map((workout) => {
-          return (
-            <Workout name={workout.name} description={workout.description} />
-          );
-        })}
-      </div>
+    <div className="App" style={{ backgroundColor: "lightgrey", minHeight: "98vh" }}>
```

Comment:
Also, two questions here :)

1: What was the decision behind setting the minHeight to 98vh? 
2: What was the decision to use inline styles and not external css files :)

Reply:
1. the 98vh was to make sure I add everything shown on 1 page. Temporary solution, but definitely not the final solution.
2. Inline styles works well with components, but external css files are better for global changes. You'll also notice that I have a theme file. Material-UI is really big on JSS (JavaScript Styling), which all styles are coded in objects rather than regular css files. If you have a better way though, I'm open to hear!